### DISCUSSION TITLE
List available scenes?

### DISCUSSION THREAD

**Discussion by Harald-U** [Koenkk/zigbee2mqtt#6930] [Devices]
In https://www.zigbee2mqtt.io/information/scenes.html are instructions to add/store, recall, and remove scenes.

Is there a way to list scenes that have been added/stored? Besides parsing out the database.db file ...

This would be especially helpful as it seems that add/store doesn't result in any meaningful response on the bridge/log topic: when I add a scene to an IKEA LED light bulb, response seems to be the status of the bulb ...

Thanks, Harald

**Comment by sjorge** [comment]
Currently there is not. There is also nothing in the ZCL to get all scenes a device is part of, there is however a View Scene that we don't implement.

I agree that it would be nice to have a scene management endpoint similar to the bindings with request/response setup.
But currently scene support is implemented as converters and not as a bridge extension. It's a lot of work to rewrite it.

**Reply by Harald-U** [reply]
Thanks, Jorge, that makes sense!
